app/assets/stylesheets/partials/_navigation.css.scss

```diff
@@ -0,0 +1,17 @@
+nav.navbar {
+  @include clearfix;
+  position: relative;
```

Comment:
Properties should be ordered border-bottom, clear, cursor, line-height, margin, overflow, position, text-align, width